### ISSUE TITLE
Install OnionScan.

### ISSUE THREAD

**Issue by AC1003** [s-rah/onionscan#175]
Hello there. I don't know how I am supposed to download the software, or how to run it.
must i have Tor first? How do i download dependencies?
I have no idea what to do.

**Comment by StasonJatham** [comment]
Yes, you have to download Tor first and have it running. The easiest way is probably to just run Tor browser https://www.torproject.org and open it. That should start the Tor service on port 9050. 
On how to download and run this project.. I think the README is pretty good I mean you literally just do a go install and you're good to go.

**Comment by VityaSchel** [comment]
Hello there!! I'm too frustrated and I decided to keep instruction here in this issue. I think maintainers should make instructions more user-friendly.
1. Install Go from official website. You can install it on windows, mac and linux. Make sure you configured it correctly and you know where $GOPATH is (this is env variable that is on ~/go on mac)
2. Then Install all dependencies. Just run `go get x` where x is url of dependency:
golang.org/x/net/proxy - For the Tor SOCKS Proxy connection.
golang.org/x/net/crypto - For PGP parsing
golang.org/x/net/html - For HTML parsing
github.com/rwcarlsen/goexif - For EXIF data extraction.
github.com/HouzuoGuo/tiedot/db - For crawl database.
For example `go get golang.org/x/net/proxy`. There will be error for golang.org/x/net/crypto, ignore it
3. Now clone this repo. You can either download it as zip or run `git clone https://github.com/s-rah/onionscan`. Other options such as github desktop are also available.
4. CD into cloned folder (onionscan). And run `go install github.com/s-rah/onionscan@latest`
5. There will be a lot of "go: finding module for package" and "go: found" messages. Ignore and wait for end.
6. Go to $GOPATH folder. It's a folder where Golang installs all dependencides. On mac os go to ~/go
7. Finally, run the executable. I don't know how to do that on Windows, but on MacOS (and I think for linux it's the same) i just typed "./onionscan" to console and got help message.
Use "./onionscan helloworld.onion" and other commands as in documentation :)

**Comment by oxosetu-png** [comment]
Chris contract tracking IP address
